packages/common/src/events/HubEvent.ts

```diff
@@ -116,11 +116,8 @@ export class HubEvent
     if (this.isDestroyed) {
       throw new Error("HubEvent is already destroyed.");
     }
-    const { deleteEvent } = await import("./api/events");
-    await deleteEvent({
-      eventId: this.entity.id,
-      ...this.context.hubRequestOptions,
-    });
+    const { deleteHubEvent } = await import("./edit");
+    await deleteHubEvent(this.entity.id, this.context.userRequestOptions);
```

Comment:
We should additionally update the tests for the `delete` method on this class to verify that it's calling the new `deleteHubEvent` fn you added below rather than the `deleteEvent` method in `common/src/events/api/events.ts`.